Formula/elasticsearch-full.rb

```diff
@@ -102,12 +102,21 @@ def plist
     port = server.addr[1]
     server.close
 
+    mkdir testpath/"config"
+    cp etc/"elasticsearch/jvm.options", testpath/"config"
+    cp etc/"elasticsearch/log4j2.properties", testpath/"config"
+
+    ENV["ES_PATH_CONF"] = testpath/"config"
```

Comment:
The main change here is moving this up to set its value for both tests, not only the second one where we use a custom configuration file.